frontend/components/Blocks/SectionBlock/SectionBlock.tsx

```diff
@@ -1,8 +1,10 @@
-import { useState, useEffect } from "react";
+import { useState, useEffect, useMemo } from "react";
 import ImageSlider from "@/components/InteractiveImage/ImageSlider";
 import RawHtml from "@/components/RawHtml/RawHtml";
 import InteractiveInputs from "@/components/InteractiveInputs/InteractiveInputs";
 import { getGrid } from "services/grid";
+import debounce from "lodash.debounce";
```

Comment:
I believe the Lodash developers [now discourage the use of the per-method packages](https://lodash.com/per-method-packages) and prefer that people use the main package instead.

That is to say, add the "lodash" package instead of "lodash.debounce" and import only what you need:

```ts
import { debounce } from "lodash";
```